frontend/app/model/MembershipSummary.scala

```diff
@@ -0,0 +1,17 @@
+package model
+
+import org.joda.time.DateTime
+
+case class MembershipSummary(startDate: DateTime,
+                             firstPaymentEndDate: DateTime,
+                             amountPaidToday: Float,
+                             planAmount: Float,
+                             nextPaymentPrice: Float,
+                             nextPaymentDate: DateTime,
+                             renewalDate: DateTime,
+                             initialFreePeriodOffer: Boolean) {
+
+
+  val annual = startDate.plusYears(1) == renewalDate
```

Comment:
I wonder if we will ever disagree with Zuora what a year is?

For instance, if someone is renewing a year after Feb 29th, will we differ on whether the `renewalDate` is Feb 28th or March 31st? This check could maybe be `math.round(Duration(startDate, renewalDate).millis.toFloat/1.year.millis) ==1`, to allow a tolerance?